capsules/src/lib.rs

```diff
@@ -63,6 +63,7 @@ pub mod proximity;
 pub mod rf233;
 pub mod rf233_const;
 pub mod rng;
+pub mod ros;
```

Comment:
```suggestion
pub mod read_only_syscall;
```

I think the same idea holds for the filename on spelling this out